test/integration/commands/xmlupload/test_resource_creation.py

```diff
@@ -72,27 +72,28 @@ def test_one_resource_without_links(ingest_client_mock: AssetClient) -> None:
     post_responses = [{"@id": "foo_1_iri", "rdfs:label": "foo_1_label"}]
     con.post = Mock(side_effect=post_responses)
     project_client = ProjectClientStub(con, "1234", None)
+
     clients = dsp_tools.commands.xmlupload.models.upload_clients.UploadClients(
         ingest_client_mock, project_client, ListClientMock()
     )
-
     xmlupload._upload_resources(clients, upload_state)
 
     assert len(con.post.call_args_list) == len(post_responses)
-    match con.post.call_args_list[0].kwargs:
-        case {
-            "route": "/v2/resources",
-            "data": {
-                "@type": "my_onto:foo_1_type",
-                "rdfs:label": "foo_1_label",
-                "knora-api:attachedToProject": {"@id": "https://admin.test.dasch.swiss/project/MsOaiQkcQ7-QPxsYBKckfQ"},
-                "@context": dict(),
-                "my_onto:hasSimpleText": [{"@type": "knora-api:TextValue", "knora-api:valueAsString": "foo_1 text"}],
-            },
-        }:
-            assert True
-        case _:
-            pytest.fail("POST request was not sent correctly")
+    post_call_args = con.post.call_args_list[0].kwargs
```

Comment:
The way the assertions were written, it was really difficult to find the error.
The difference now compared to before is that the absolute IRIs are used.